valio-master/Valio/Images.xcassets/LaunchImage.launchimage/Contents.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Not sure how old is this, but if we have a `UINavigationController` as a root view controller a Launch storyboard would be more maintainable, as we just put the image in the bar and we're done and it scales to all devices.